CMakeLists.txt

```diff
@@ -0,0 +1,181 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(Atlas)
+
+option(PRELOAD_MAP "Preload default map." OFF)
+
+if (PRELOAD_MAP)
+  add_definitions(-DPRELOAD_MAP)
+endif()
+
+set(DIST_PATH ${PROJECT_BINARY_DIR}/dist)
+set(DIST_PLUGINS_PATH ${DIST_PATH}/plugins)
+set(DIST_LIB_PATH ${DIST_PATH}/lib)
+set(DIST_RESOURCES_PATH ${DIST_PATH}/res)
```

Comment:
The folder renaming is not mentioned elsewhere. All resource path in the code refers to /resources. Maybe it's better to leave it unchanged?
`set(DIST_RESOURCES_PATH ${DIST_PATH}/resources)`